core/input/gesturehandler.js

```diff
@@ -186,14 +186,14 @@ export default class GestureHandler {
         }
 
         if (!this._hasDetectedGesture()) {
-            // Ignore moves smaller than the minimum threshold
-            if (Math.hypot(deltaX, deltaY) < GH_MOVE_THRESHOLD) {
-                return;
-            }
 
-            // Can't be a tap or long press as we've seen movement
-            this._state &= ~(GH_ONETAP | GH_TWOTAP | GH_THREETAP | GH_LONGPRESS);
-            this._stopLongpressTimeout();
+            let deltaMove = Math.hypot(deltaX, deltaY);
+
+            // Can't be a tap or long press if we've seen movement
+            if (deltaMove >= GH_MOVE_THRESHOLD) {
+                this._state &= ~(GH_ONETAP | GH_TWOTAP | GH_THREETAP | GH_LONGPRESS);
+                this._stopLongpressTimeout();
+            }
 
             if (this._tracked.length !== 1) {
                 this._state &= ~(GH_DRAG);
```

Comment:
This should have already been excluded in the touchstart handler. Perhaps best to remove this redundancy to make things easier to read and understand.